tests/tck/features/delete/DeleteTag.feature

```diff
@@ -0,0 +1,287 @@
+      | VertexID | bachelor.name | bachelor.speciality |
+    When executing query:
+      """
+      LOOKUP ON player WHERE player.name == "Tim Duncan"
+      """
+    Then the result should be, in any order:
+      | VertexID |
+
+  Scenario: delete string vid multiple vertex one tag
+    Given an empty graph
+    And load "nba" csv data to a new space
+    # before delete tag
+    When executing query:
+      """
+      FETCH PROP ON player "Tim Duncan" YIELD player.name, player.age
+      """
+    Then the result should be, in any order:
+      | VertexID     | player.name  | player.age |
+      | "Tim Duncan" | "Tim Duncan" | 42         |
+    When executing query:
+      """
+      FETCH PROP ON player "Tony Parker" YIELD player.name, player.age
+      """
+    Then the result should be, in any order:
+      | VertexID      | player.name   | player.age |
+      | "Tony Parker" | "Tony Parker" | 36         |
+    When executing query:
+      """
+      LOOKUP ON player WHERE player.name == "Tim Duncan"
+      """
+    Then the result should be, in any order:
+      | VertexID     |
+      | "Tim Duncan" |
+    When executing query:
+      """
+      LOOKUP ON player WHERE player.name == "Tony Parker"
+      """
+    Then the result should be, in any order:
+      | VertexID      |
+      | "Tony Parker" |
+    # delete one tag
+    When executing query:
+      """
+      DELETE TAG player FROM "Tim Duncan", "Tony Parker";
+      """
+    Then the execution should be successful
+    # after delete tag
+    When executing query:
+      """
+      FETCH PROP ON player "Tim Duncan" YIELD player.name, player.age
+      """
+    Then the result should be, in any order:
+      | VertexID | player.name | player.age |
+    When executing query:
+      """
+      FETCH PROP ON player "Tony Parker" YIELD player.name, player.age
+      """
+    Then the result should be, in any order:
+      | VertexID | player.name | player.age |
+    When executing query:
+      """
+      LOOKUP ON player WHERE player.name == "Tim Duncan"
+      """
+    Then the result should be, in any order:
+      | VertexID |
+    When executing query:
+      """
+      LOOKUP ON player WHERE player.name == "Tony Parker"
+      """
+    Then the result should be, in any order:
+      | VertexID |
+
+  Scenario: delete string vid from pipe
+    Given an empty graph
+    And load "nba" csv data to a new space
+    # before delete tag
+    When executing query:
+      """
+      GO FROM "Tim Duncan" OVER serve YIELD serve._dst as id
+      """
+    Then the result should be, in any order:
+      | id      |
+      | "Spurs" |
+    When executing query:
+      """
+      FETCH PROP ON team "Spurs" YIELD team.name
+      """
+    Then the result should be, in any order:
+      | VertexID | team.name |
+      | "Spurs"  | "Spurs"   |
+    # delete one tag
+    When executing query:
+      """
+      GO FROM "Tim Duncan" OVER serve YIELD serve._dst as id | DELETE TAG team FROM $-.id
```

Comment:
Test following cases:

```
GO FROM "Tim Duncan" OVER serve YIELD serve._dst as id | DELETE TAG team FROM $-.id, "Lakers"
$var = GO FROM "Tim Duncan" OVER serve YIELD serve._dst as id; DELETE TAG team FROM $var.id, "Lakers"
```

Reply:
If not support above cases, please report error and add these scenarios in TCK

Reply:
🆗 , done